src/engine/ruleset/decoders/auditd/auditd-kv.yml

```diff
@@ -26,7 +26,7 @@ parse:
 normalize:
   - map:
       - auditd.log: +ef_merge/$~kv_map
-      - auditd.log.node: +ef_rename/$~node
+      - ~auditd.log.node: +ef_rename/$~node
```

Comment:
Why has this field been changed to a temporal one?

Reply:
Going back to the pipeline, taken as a base for the decoder, that field is used and mapped as in this case, but I couldn't find any description of it or any other use in the schemas. IMHO It wasn't a wrong approach to define it as temporary, but if it is needed after, we could create a custom field with that name.